python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -183,7 +183,11 @@ def get_assets_latest_info(
         asset_key: graphene_info.context.asset_graph.get(asset_key) for asset_key in asset_keys
     }
 
-    asset_records = AssetRecord.blocking_get_many(graphene_info.context, asset_keys)
+    asset_records = [
+        record
+        for record in AssetRecord.blocking_get_many(graphene_info.context, asset_keys)
+        if record
```

Comment:
`if record is not None` pls -- it's definitely exactly the same in this particular case but I believe banishing this sort of implicit None check from our codebase would be good in principle (and would have saved us from some nasty `set()` vs `None` behavior difference issues in the past)

Reply:
ah yes, good call